src/object/srv_ec_aggregate.c

```diff
@@ -35,15 +35,37 @@
  *	- The parity for peer parity extents is transferred.
  *	- Replicas for the stripe are removed from parity targets.
  *
+ * If replicas are partial, and prior parity exists:
+ *	- If less than half cells are updated (have replicas, parity is updated:
+ *		- Old data cells for cells with replica data are fetched from
+ *		  data targets (old, since fetched at epoch of existing parity).
+ *		- Peer parity is fetched.
+ *		- Parity is incrementally updated.
```

Comment:
just note that this kind of incremental parity update was not used/verified in our code before, but assume should work based on correct usage of ISAL.

Reply:
I have unit tests, and there is also verification in the daos_test file (daos_aggregate_ec.c) that confirms the correctness of the incremental update.